lib/CodeBuilder/Tests/Unit/Domain/TemplatePathResolver/FilterPhpVersionDirectoryIteratorTest.php

```diff
@@ -56,7 +57,7 @@ private function createFakeDirectory(string $filename): SplFileInfo
 
     private function createFakeSplFileInfo(string $filename, bool $isDir = false): SplFileInfo
     {
-        $file = $this->prophesize(SplFileInfo::class);
+        $file = $this->prophesize(\Symfony\Component\Finder\SplFileInfo::class);
```

Comment:
not sure why this change? it should not be the FQN - should it be the Symfony SPL info?

Reply:
This is the class that the Prophecy spltfileinfo patch checks against here: https://github.com/phpspec/prophecy/blob/master/src/Prophecy/Doubler/ClassPatch/SplFileInfoPatch.php#L119

Reply:
Notice that the return type is still `\SplFileInfo` because we're only using the symfony one to avoid the deprecation. Apparently there is an internal implementation detail about `\SplFileInfo` that makes it difficult for Prophecy to prophesize it without arguments.